db_tasks.py

```diff
@@ -99,20 +102,36 @@ def do_schema_dump(outfile, cld_url, export_url, username, password, service_bin
 
     resp = conn.getresponse()
     if resp.status != 200:
-        raise Exception('Could not get schema from {}'.format(export_url))
+        raise Exception('Could not get schema from {}'.format(dbexport_host))
     schema = resp.read()
     conn.close()
 
     with open(outfile, "w") as schema_file:
         schema_file.write(schema)
 
 
-def test_migration(shelf_filename, cld_url, export_url, username, password, service_binding):
+# cfapi_host: api.system.example.com
+# service_name: database name (must be at least one app already deployed that is bound to it)
+def v2_test_migration(shelf_filename, cfapi_host, username, password, service_name):
```

Comment:
you’ve cut back on env vars but made the script more confusing (I am thinking of the next person that tries to figure this out).  At least there's comments.

Reply:
yes, given that both `cld_url` and `export_url` are derived from the API address, it seemed annoying to need to specify over and over.

I also changed the `_url` bits to `_host` to more accurately describe what they are meant to be - I think this has tripped us up in the past.

I think after prod is updated, we can remove the original `test_migration` - I left it in for now so as not to break the prod CI.